src/main/java/edu/sfsu/db/CampusTestDB.java

```diff
@@ -1,13 +1,28 @@
 package edu.sfsu.db;
 
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 
 public class CampusTestDB extends DB implements CampusDB {
 
+    final static protected String DB_NAME = "ADVISING";
```

Comment:
I am a little bit confused. Those changes you have here were already in the previous PR. Can you please check and remove changes that were already merged? Thx.

Reply:
Ah I think I just forgot to push the merge with the previous master. I pushed the merge and the repeated section of the campusTestDb disappeared.

Reply:
There are two commits now. Please rebase to master, squash into one commit, then force-push. Thx.